src/avkit.cs

```diff
@@ -549,5 +553,25 @@ interface AVRoutePickerViewDelegate {
 		[Export ("routePickerViewDidEndPresentingRoutes:")]
 		void DidEndPresentingRoutes (AVRoutePickerView routePickerView);
 	}
+
+	[TV (11,2), NoiOS, NoMac, NoWatch]
+	[BaseType (typeof (NSObject))]
```

Comment:
since there's no way you can set an `AVDisplayManager` to a `UIWindow` I think it's safer to assume (even if the header are not clear) that developers should not be directly creating instances of the type -> `[DisableDefaultCtor]`